rpc/backend/call_tx_test.go

```diff
@@ -297,15 +297,15 @@ func (suite *BackendTestSuite) TestSendRawTransaction() {
 			common.Hash{},
 			false,
 		},
-		{
+		/*{
```

Comment:
Is there any reason for pass?

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/e2cc87462bed5906b75418652bd0d54437947eaa